fineract-core/src/main/java/org/apache/fineract/infrastructure/dataqueries/data/ResultsetColumnHeaderData.java

```diff
@@ -239,40 +206,58 @@ private DisplayType calcDisplayType() {
     }
 
     public static DisplayType calcColumnDisplayType(JdbcJavaType columnType) {
+        return switch (getJdbcType(columnType)) {
```

Comment:
I see no any value of this changes....
If you want to change the IF conditions to switch fine... but why to map JdbcJavaType to String and from String to DisplayType?

Map JdbcJavaType to DisplayType with mapper if you want...